### PR TITLE
allow adding multiple msp users; add new msp user for enikshay

### PR DESCRIPTION
**Pull request by mkangia**
Building on https://github.com/dimagi/commcarehq-ansible/pull/938/files
1. to allow multiple users to be added
2. to add a new user for enikshay

### CODE REVIEW

File: ansible/vars/enikshay/unx40423.pub [hunk 1]
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDvNQLnUdNc+Y6gk+IcYjeasQ7hyCFn+OFBeNfe5hTZvAnOEPDzstTrs+SOcNLa/Qoc7XxfxIQBWKFkQUrJ17vKmm6suE6rXO6MmfAtAsxSQMPPTc6BjakraVncP0OcnG6qVV6P+r0HQJTicBMSU08heTsZfxBXIQbt39bjcjJiaA4E3wF57cgitcz0AEfmi9L5ah+0L2dST3FCHWqU9uXbdHE560PHW6PhXfisEx7F7XQahsniwOpT7dkmxaJKL3Pm7QTZdatUrJ5TceJxYisybxUKzmGDBuAONtIgcJ+dSRxiZZqlqu/rq8EwPpOYhndFafKm28u9EuhG16Y21Q/X unx40423@ubuntu
```
Comment (javierwilson) [inline ansible/vars/enikshay/unx40423.pub]: we don't need this, i think

**Comment by javierwilson** [comment]
looks good to me! also, thanks this make things more flexible for other clients too.

File: ansible/vars/enikshay/unx40423.pub (hunk 1 above)
Comment (mkangia) [reply ansible/vars/enikshay/unx40423.pub]: @javierwilson this is the different pub key that should be added for ssh for new user `unx40423` like we had for msp?

Comment (javierwilson) [reply ansible/vars/enikshay/unx40423.pub]: got it. yeah we keep those in files.

**Pull request merged**